x-pack/solutions/security/plugins/session_view/public/components/detail_panel_alert_list_item/styles.ts

```diff
@@ -15,8 +15,8 @@ export const useStyles = (minimal = false, isInvestigated = false) => {
   const cached = useMemo(() => {
     const { colors, font, size, border } = euiTheme;
 
-    const dangerBorder = transparentize(colors.danger, 0.2);
-    const dangerBackground = transparentize(colors.danger, 0.08);
+    const dangerBorder = transparentize(colors.danger, 0.2); // TODO: replace transparentize with color token
```

Comment:
here and in pretty much every place in `session_view` we need to replace `transparentize` and other color functions with color tokens. But it requires input from designers, on which tokens to use exactly